starter-code/lib/harryPotter.js

```diff
@@ -8,3 +8,24 @@
                    "4 October", "Filius Flitwick", "17 October", "Molly Weasley",
                    "30 October", "Bill Weasley", "29 November", "Rubeus Hagrid",
                    "6 December", "Charlie Weasley", "12 December"];
+
+var moreBirthdays = ["Lily Evans", "30 January", "James Potter", "27 March",
+                     "Dudley Dursley", "30 June", "Tom Riddle", "31 December"];
+
+var createPairs = function(arr) {
+    var pairArray = [];
+  
+    while(arr.length) {
+        pairArray.push(arr.splice(0,2));
+    }
+    
+    return pairArray;
+}
```

Comment:
you could use the chunk method:

var pairsOfBirthdays = _.chunk(birthdays, 2);